webpack.config.ts

```diff
@@ -470,14 +473,24 @@ const appConfig: Configuration = {
       chunks: 'async',
       maxInitialRequests: 10, // (default: 30)
       maxAsyncRequests: 10, // (default: 30)
-      cacheGroups: {
-        ...localeChunkGroups,
-      },
+      cacheGroups,
     },
 
-    // This only runs in production mode
-    // Grabbed this example from https://github.com/webpack-contrib/css-minimizer-webpack-plugin
-    minimizer: ['...', new CssMinimizerPlugin()],
+    minimizer: [
+      new TerserPlugin({
+        parallel: true,
+        minify: TerserPlugin.esbuildMinify,
+      }),
+      new CssMinimizerPlugin({
+        parallel: true,
+        minify: CssMinimizerPlugin.lightningCssMinify,
+        minimizerOptions: {
+          targets: lightningcss.browserslistToTargets(
+            browserslist(packageJson.browserslist.production)
+          ),
+        } as never,
```

Comment:
Why did you have to cast it?

Reply:
`minimizerOptions.targets` is lighhtningCss specific, but I found a solution 👍